### ISSUE TITLE
BDD Process diagram

### ISSUE THREAD

**Issue by aslakhellesoy** [cucumber/docs#136]
To make it easier to understand who does what when and why. And how it fits into e.g. Scrum

**Comment by mgundala** [comment]
Hi Aslak,
Please let me know if I am wrong about this issue,
Are we defining the responsibilities of each personalities in the Methodologies e.g. Scrum Methodology and Explain how BDD Suites  in every aspect?

**Comment by aslakhellesoy** [comment]
Yes that is the intent. We have this diagram in our training material and will add it to the website.

**Comment by mlvandijk** [comment]
@aslakhellesoy Could you send add this diagram, or send it to me so I can add it?

**Comment by aslakhellesoy** [comment]
Hey @sebrose - could you point me to the diagram I refer to above?

@mgundala it won't be prescriptive or aim to cover all contexts and teams. It's a starting point, and readers will be encouraged to find a configuration that works in their context. We'll also stress the importance of incremental process change to maximise the chances of making desired changes.
